serverless/functions/sessionUtil.ts

```diff
@@ -0,0 +1,89 @@
+import jwt from "jsonwebtoken";
+
+const authorizerSecret = process.env.AUTHORIZER_SECRET || "";
+
+const handleSessionToken = async (
+  sessionToken: string,
+  userId: string
+): Promise<{ sessionId?: string; error?: string; statusCode?: number }> => {
+  // handle sessionToken
+  let currentToken: {
+    status: string;
+    sessionId: string;
+    exp: number;
+    participants: [string];
+  };
+  let sessionId = "";
+  if (authorizerSecret != "") {
+    console.log("session support enabled");
+    try {
+      currentToken = jwt.verify(sessionToken, authorizerSecret + "", {
+        algorithms: ["HS512"],
+        ignoreExpiration: true
+      }) as {
+        sessionId: string;
+        status: string;
+        exp: number;
+        participants: [string];
+      };
+    } catch (err) {
+      console.error(
+        "invalid session token. userId=" + userId + ", error=" + err
+      );
+      return {
+        statusCode: 401,
+        error: "INVALID_TOKEN"
+      };
+    }
+
+    // TODO: add expiration grace period
+    if (Date.now() / 1000 > currentToken.exp) {
+      console.error(
+        "expired token. userId=" +
+          userId +
+          ", sessionId=" +
+          currentToken.sessionId
+      );
+      return {
+        statusCode: 401,
```

Comment:
`403` would be a better statusCode here, `401` in the app side always logs out the user and goes to login page